hazelcast/src/test/java/com/hazelcast/cardinality/CardinalityEstimatorStressTest.java

```diff
@@ -43,8 +43,8 @@ public void addBigRange_checkErrorMargin_completeWithinSixteenMins() {
         // timeout helps to make sure the encoding engine used is the correct one.
         long expected = 10 * 1000 * 1000;
 
-        float tolerancePct = .3f;
-        long acceptableDelta = (long) ((tolerancePct / 100) * expected);
+        float sigma = 0.008125f;
+        long acceptableDelta = (long) (2 * sigma * expected);
```

Comment:
Do we need to multiply the `sigma` by 2 here?
By the way, I don't understand where the previous value (`tolerancePct = .3f`) comes from.

Reply:
Previous value is just a number very close to the actual number IMO. We can also do a similar thing for little endian too, but I didn't. I multiplied by two, because the actual number has ~0.01 delta. So one sigma wasn't enough.

Reply:
Thanks, Ramiz for the explanation!